### PR TITLE
User can start the car.

### PR DESCRIPTION
**Pull request by gitrkaplan**

### CODE REVIEW

**Comment by gitrkaplan** [comment]
driving_game_start

File: main.js [hunk 1]
```diff
@@ -1,5 +1,93 @@
...
+        break
+      case 'West':
+        this.location[0] -= this.speed
+    }
+    sportsCar.style.left = this.location[0] + 'px'
+    sportsCar.style.top = this.location[1] + 'px'
+  }
+}
+
+function onKeyDown(event) {
```
Comment (thebearingedge) [inline main.js:49]: Don't include code that isn't for the current feature.

File: main.js [hunk 2]
```diff
@@ -1,5 +1,93 @@
...
+    const [x, y] = location
+    carType.style.left = x + 'px'
+    carType.style.top = y + 'px'
+  }
+  static start(car) {
+    intervalId = setInterval(function () {
+      car.move()
+    }, 16)
+  }
+  turn(direction) {
```
Comment (thebearingedge) [inline main.js]: This is extra code.

File: main.js [hunk 3]
```diff
@@ -1,5 +1,93 @@
...
+    this.direction = direction
+  }
+  accelerate(amount) {
```
Comment (thebearingedge) [inline main.js]: This is extra code.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please fix the Pull Request title and address the notes.

**Comment by gitrkaplan** [comment]
@thebearingedge The requested changes have been made.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**